superset-frontend/src/dashboard/components/FiltersBadge/index.tsx

```diff
@@ -38,6 +37,7 @@ import {
   DashboardLayout,
   RootState,
 } from '../../types';
+import { setDirectPathToChild } from '../../actions/dashboardState';
```

Comment:
pure subjective and personal preference, but double `..` always makes me cringe 😬 
```suggestion
import { setDirectPathToChild } from 'src/dashboards/actions/dashboardState';
```